Sources/BraintreePayPal/BTPayPalError.swift

```diff
@@ -32,7 +32,10 @@ public enum BTPayPalError: Error, CustomNSError, LocalizedError, Equatable {
 
     /// 9. Deallocated BTPayPalClient
     case deallocated
-    
+
+    /// 10. Unknown app switch error occurred.
+    case unknownAppSwitchError
```

Comment:
Maybe we can specify that this error is specific to a valid redirect URL that is missing the `cancel` or `success` paths?

Reply:
That's a good call, updated here: https://github.com/braintree/braintree_ios/pull/1237/commits/2a1d7d79993eaabe6f93a199fe31441e8653bf73